R/utils.R

```diff
@@ -374,3 +363,54 @@ select_decorators <- function(decorators, scope) {
     list()
   }
 }
+
+#' Set the attributes of the last chunk outputs
+#' @param data (`teal_card`) object to modify.
+#' @param attributes (`list`) of attributes to set on the last chunk outputs.
+#' @param n (`integer(1)`) number of the last element of `teal_card` to modify.
+#' it will only change `chunk_output` objects.
+#' @param inner_classes (`character`) classes within `chunk_output` that should be modified.
+#' This can be used to only change `recordedplot`, `ggplot2` or other type of objects.
+#' @keywords internal
+modify_last_chunk_outputs_attributes <- function(teal_card,
```

Comment:
maybe something shorter? `set_card_attributes` or `set_card_attrs`
```suggestion
set_card_attributes<- function(teal_card,
```